static/js/public/fsf-expandable-yaml-details.js

```diff
@@ -0,0 +1,21 @@
+export default function initExpandableYaml() {
+  const showMoreContainer = document.querySelectorAll(
+    "[data-js='js-show-more']"
+  );
+
+  if (showMoreContainer) {
+    showMoreContainer.forEach(el => {
+      const fadeEL = el.querySelector(".p-show-more__fade");
+      const linkEl = el.querySelector(".p-show-more__fade");
```

Comment:
Should this be the same selector? Maybe `.p-show-more__link`?

Reply:
well spotted!